generate-repo-stub-readme.sh

```diff
@@ -30,8 +34,36 @@ This is the Git repo of the Docker [official image](https://docs.docker.com/dock
 The full readme is generated over in [docker-library/docs]($gitRepo), specifically in [docker-library/docs/$repo]($gitRepo/tree/master/$repo).
 
 See a change merged here that doesn't show up on the Docker Hub yet? Check [the "library/$repo" manifest file in the docker-library/official-images repo](https://github.com/docker-library/official-images/blob/master/library/$repo), especially [PRs with the "library/$repo" label on that repo](https://github.com/docker-library/official-images/labels/library%2F$repo). For more information about the official images process, see the [docker-library/official-images readme](https://github.com/docker-library/official-images/blob/master/README.md).
+EOREADME
+
+badges=()
+
+n=$'\n'
 
-[![Travis CI](https://img.shields.io/travis/$travisRepo/master.svg)](https://travis-ci.org/$travisRepo/branches)
+travisImage="https://img.shields.io/travis/$travisRepo/master.svg"
+if wget -q --spider "$travisImage" &> /dev/null; then
```

Comment:
Note that Travis doesn't actually return 404 for repos that aren't set up -- they return a valid SVG.  Not sure how to resolve that, but it isn't a big deal for our current needs (since it's a problem we've already got).